papermill/execute.py

```diff
@@ -231,5 +238,11 @@ def raise_for_execution_errors(nb):
                     traceback=output.traceback
                 )
             break
+
     if error:
+        # Write notebook back out with the Error Message at the top of the Notebook.
+        error_msg = ERROR_MESSAGE_TEMPLATE % str(error.exec_count)
+        error_msg_cell = nbformat.v4.new_markdown_cell(source=error_msg)
+        nb.cells = [error_msg_cell] + nb.cells
+        write_ipynb(nb, output_path)
```

Comment:
This is an error from their execution or from papermill? I'm confused.

Reply:
This is an error encountered by the running notebook. I can update the var name to make that more clear if you'd like.

Reply:
What I'd probably write (in the template above) is:

```
This notebook encountered an exception at `In [%s]` while being run by papermill.
```

Reply:
What I'm a bit lost on is why we're not embedding the classic error / stacktrace within the notebook where it errored (or perhaps it does that anyways...).